tools/deployment/pytorch2onnx.py

```diff
@@ -118,11 +119,12 @@ def pytorch2onnx(model,
         input_dic = {'input': imgs.detach().cpu().numpy()}
         input_shape_dic = {'input': list(input_shape)}
 
-        onnxsim.simplify(
-            output_file,
-            input_shapes=input_shape_dic,
-            input_data=input_dic,
-            dynamic_input_shape=dynamic_export)
+        model_opt, check_ok =onnxsim.simplify(
+           output_file,
+           input_shapes=input_shape_dic,
+           input_data=input_dic,
+           dynamic_input_shape=dynamic_export)
+        onnx.save(model_opt, output_file)
```

Comment:
Please check the `check_ok` result.
```suggestion
        if check_ok:
            onnx.save(model_opt, output_file)
            print(f'Successfully simplified ONNX model: {output_file}')
        else:
            print(f'Failed to simplify ONNX model, the simplified model could not be validated.')
```

Reply:
And please fix the lint. Refers to our [CONTRIBUTING.md](https://github.com/open-mmlab/mmclassification/blob/master/.github/CONTRIBUTING.md) to set up pre-commit lint hooks.